src/templates/macros/pageheader.html

```
@@ -0,0 +1,27 @@
+    literally in the HTML of every shared-layout page. -#}
+<header class="pageheader" role="banner">
+    <div class="pageheader-main">
+        {% if eyebrow %}<div class="eyebrow">{{ eyebrow }}</div>{% endif %}
+        <h1 class="page-title{% if title_class %} {{ title_class }}{% endif %}"{% if title_attr %} title="{{ title_attr }}"{% endif %}>{{ title }}</h1>
+        {% if description %}<p class="page-desc">{{ description }}</p>{% endif %}
+        {% if chips_html %}
+        <div class="page-meta">{{ chips_html | safe }}</div>
+        {% elif chips %}
+        <div class="page-meta">
+            {% for chip in chips %}
+            {% if chip.url %}<a href="{{ chip.url }}"{% if chip.id %} id="{{ chip.id }}"{% endif %} class="status-chip{% if chip.variant %} {{ chip.variant }}{% endif %}">{{ chip.label }}</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)